src/server/pegasus_server_impl.cpp

```diff
@@ -776,6 +776,121 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
     _pfc_multi_get_latency->set(dsn_now_ns() - start_time);
 }
 
+void pegasus_server_impl::on_batch_get(batch_get_rpc rpc)
```

Comment:
This function is too long. Could you split it into several small functions?

Reply:
110 lines, seemly not long,  and i think it's simple

Reply:
And there is no split functions can be reused, I think keep it is OK.